apis/replication.storage/v1alpha1/volumegroupreplicationcontent_types.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2024 The Kubernetes-CSI-Addons Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// VolumeGroupReplicationContentSpec defines the desired state of VolumeGroupReplicationContent
+type VolumeGroupReplicationContentSpec struct {
+	// VolumeGroupreplicationRef specifies the VolumeGroupReplication object to which this
+	// VolumeGroupReplicationContent object is bound.
+	// VolumeGroupReplication.Spec.VolumeGroupReplicationContentName field must reference to
+	// this VolumeGroupReplicationContent's name for the bidirectional binding to be valid.
+	// For a pre-existing VolumeGroupReplicationContent object, name and namespace of the
+	// VolumeGroupReplication object MUST be provided for binding to happen.
+	// This field is immutable after creation.
+	// Required.
+	// +kubebuilder:validation:XValidation:rule="has(self.name) && has(self.__namespace__)",message="both volumeGroupReplicationRef.name and volumeGroupReplicationRef.namespace must be set"
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="volumeGroupReplicationRef is immutable"
+	VolumeGroupReplicationRef corev1.ObjectReference `json:"volumeGroupReplicationRef"`
+
+	// VolumeGroupReplicationHandle is a unique id returned by the CSI driver
+	// to identify the VolumeGroupReplication on the storage system.
+	VolumeGroupReplicationHandle string `json:"volumeGroupReplicationHandle"`
+
+	// provisioner is the name of the CSI driver used to create the physical
+	// volume group on
+	// the underlying storage system.
+	// This MUST be the same as the name returned by the CSI GetPluginName() call for
+	// that driver.
+	// Required.
+	Provisioner string `json:"provisioner"`
+
+	// VolumeGroupReplicationClassName is the name of the VolumeGroupReplicationClass from
+	// which this group replication was (or will be) created.
+	// +optional
+	VolumeGroupSnapshotClassName string `json:"volumeGroupReplicationClassName"`
```

Comment:
```suggestion
	VolumeGroupReplicationClassName string `json:"volumeGroupReplicationClassName"`
```

Reply:
Thanks for catching it. https://github.com/csi-addons/kubernetes-csi-addons/pull/594 here is the fix, feel free to send PR's if the PR's are already merged/closed.